plugin.py

```diff
@@ -46,5 +47,51 @@ def on_pre_server_command(self, params: Mapping[str, Any], done_callback: Callab
             done_callback()
         return True
 
+    def on_server_response_async(self, method: str, response: Response) -> None:
+        if method == 'codeAction/resolve':
+            self.maybe_handle_move_code_action(cast(CodeAction, response.result))
 
+    def maybe_handle_move_code_action(self, code_action: CodeAction) -> None:
+        if code_action['title'] != 'Move Function':
+            return
+        session = self.weaksession()
+        if not session:
+            return
+        command = cast(MoveFunctionCommand, code_action.get('command'))
+        if not command:
+            return
+        _, params, function_name = command.get('arguments')
+        move_params = {
+            'sourceUri': params.get('textDocument').get('uri'),
+            'params': params
+        }  # type: MoveParams
+        get_move_destinations_request = Request('elm/getMoveDestinations', move_params)
+        session.send_request(get_move_destinations_request,
+                             lambda res: self.on_get_destinations(res, params, function_name))
```

Comment:
I wouldn't create variable for the `Request` since it doesn't add any clarity over directly using `Request`.